build/legacy.mk

```diff
@@ -4,6 +4,31 @@ test-te: test-server
 # test-ee used to just run the enterprise edition tests, but now runs whatever is available
 test-ee: test-server
 
+## Runs govet against all packages. This is now subsumed by make golangci-lint.
```

Comment:
Apologies, @agnivade -- instead of just replicating the old functionality, I was wondering we should just alias the new functionality, e.g.:
```
govet: golangci-lint
```

or whatever the right target steps are. I don't think we need to preserve exactly the old code, just the interface to same.

Reply:
I agree. Done.

Reply:
Do we then want to make the same for `gofmt` as it's now also part of 'golangci-lint`?

Reply:
I think we should leave `gofmt` alone to do just `gofmt`ing. Perhaps we can add a line after "Running GOFMT", to indicate to the user to run `golangci-lint` instead ? That way, the user can transition to the correct command and we can remove these later.

Reply:
@lieut-data - Since you approved, just wanted to confirm that you don't want me to add anything to the "Running GOFMT" line ?

Reply:
@agnivade, I'm fine leaving as-is.